Libplanet.Store/MemoryStore.cs

```diff
@@ -78,11 +77,11 @@ void IStore.DeleteChainId(Guid chainId)
             _canonicalChainId = chainId;
 
         long IStore.CountIndex(Guid chainId) =>
-            _indices.TryGetValue(chainId, out ImmutableTrieList<BlockHash> index) ? index.Count : 0;
+            _indices.TryGetValue(chainId, out var index) ? index.Count : 0;
```

Comment:
We tend to use explicit type declaration unless it is clear from the immediate context what type we'd be getting.

Reply:
yes